internal/process/defunct_processes.go

```diff
@@ -0,0 +1,90 @@
+package process
+
+import (
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// ProcessFS is the process file system.
+const ProcessFS = "/proc"
+
+// Stat represents status information of a process from /proc/[pid]/stat.
+type Stat struct {
+	// Fname is the filename of the executable.
+	Fname string
+
+	// State is the state of the process.
+	State string
+}
+
+// DefunctProcesses returns the number of zombie processes in the node.
+func DefunctProcesses() (int, error) {
```

Comment:
nit: you can pre-declare `defunctCount` here and let's make it an uint:
```suggestion
func DefunctProcesses() (defunctCount uint, err error) {
```